.circleci/config.yml

```diff
@@ -79,14 +79,6 @@ commands:
           name: Wait for sauce connect to be up
           command: wget --retry-connrefused --no-check-certificate -T 60 localhost:4445
 
-  stop_sauce_connect:
-    description: Stop all sauce connect tunnels
-    steps:
-      -  run:
-          name: Stop sauce connect
-          command: killall sc
-
-
```

Comment:
Looks like the new images auto-stop background processes. There doesn't seem to be any need to do this; even without this, I see Sauce Connect saying:

```
Got signal interrupt
```

...and in fact, if we try to kill the process, then it will fail with an error code.